### PR TITLE
Added status and click YML

### PR DESCRIPTION
**Pull request by dpgraham**
* Renamed template.yml to example.yml
* Made some minor revisions (e.g.: made endpoint.response not required and have it show as 'null' when undefined)

### CODE REVIEW

File: commands-yml/commands/element/click.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+name: Click
+short_description: Click element at it's center point.
```
Comment (jlipps) [inline commands-yml/commands/element/click.yml]: its

File: commands-yml/commands/status.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      await driver.status();
+  javascript_wdio:
+    |
+      driver.status();
+  ruby:
+    |
+      # TODO
+  php:
+    |
+      # TODO
```
Comment (jlipps) [inline commands-yml/commands/status.yml]: php uses c-style comments

File: docs/en/commands/status.md [hunk 3]
```diff
@@ -77,20 +90,22 @@ Implementations may optionally include additional meta information as part of th
 
 ### URL Parameters
 
-None
+|name|description|
```
Comment (jlipps) [inline docs/en/commands/status.md]: probably don't want a blank table, right?

**Review by jlipps — COMMENTED** [review]
looks good, just a few remarks

File: docs/en/commands/status.md (hunk 3 above)
Comment (dpgraham) [reply docs/en/commands/status.md]: Right. I need to move the if block.

**Comment by dpgraham** [comment]
Fixes pushed

**Review by jlipps — APPROVED** [review]

**Pull request merged**